.codeclimate.yml

```diff
@@ -0,0 +1,5 @@
+#CodeClimate config file
+
+plugins:
+  swiftlint:
+    enabled: true
```

Comment:
Can we have a newline here?

Reply:
Sure! my bad

Reply:
isn't this the default? I don't see the purpose of this file

Reply:
@MaicolBen You say that enabling the plugin in the CC dashboard is the same as this?

Reply:
When you add the project to CC, it already runs for swiftlint, this file is for disabling or customizing the plugin checks.

Reply:
Ok, I will delete this file and leave the Swiftlint pluging option enabled on CC.